src/models/generated/RenewableDispatch.jl

```diff
@@ -53,6 +55,8 @@ mutable struct RenewableDispatch <: RenewableGen
     powerfactor::Float64
     "Operation Cost of Generation [`TwoPartCost`](@ref)"
     op_cost::TwoPartCost
+    "Base power of the unit in MVA"
+    machine_basepower::Float64
```

Comment:
I am not sure we should call it machine_basepower in this case since it could also be an Inverter